cidnfstack.py

```diff
@@ -650,15 +655,13 @@ def _start_commandline():  # pylint: disable=R0912,R0915
                                  ' local-build',
         action="append")
     buildparser.add_argument(
-        '-c', '--copr', type=unicode, nargs='+', default=['local-build', 'tito'], metavar=('PROJECT', 'BUILDER'),
-        help='the name of the Copr project. If option not present default values are used: "local-build" "tito" and '
-             'the test runs locally (no copr build). '
-             'If BUILDER tito, the "tito" executable must be available. '
-             'If BUILDER librepo, the "cp", "dirname", "echo", "git", "mv", "rpmbuild", "rm", "sed", "sh" and "xz" '
-             'executables must be available.'
-             'If BUILDER libscomps, the "python" and "rpmbuild" executables must be available.')
+        '-n', '--no-dnf-docker-test', help='do not start the functional test of project after Copr build',
+        action="store_true")
     buildparser.add_argument(
```

Comment:
Sorry I was inaccurate - I added -t option and reorder arguments. I can split it into two. I really want to merge both PRs.